mixin.js

```diff
@@ -351,12 +370,21 @@
         // loop through event group and trigger them
         for (i=0; i<event.length; i++)
         {
-          // detect key- events from touch- events
-          // TODO: Add support for mouse- events
-          eventObject = this[event[i].match(/^key/) ? '_createTypeEvent' : '_createTouchEvent'].apply(this, [event[i]].concat(extras));
-
-          // trigger event on target
-          target.trigger(eventObject);
+          switch(event) {
+            case 'focus':
```

Comment:
Yes, I have an idea. Going to work on it next week.

Should simplify usage of `_triggerEvents` and make it easier to extend.